.github/workflows/tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Tests
```

Comment:
@Xiaoyan-Li This file sets the repository to run the tests on pushes to the AlgebraicJulia/StockFlow.jl repo and pull requests.